### DISCUSSION TITLE
Supabase Swift Documentation?

### DISCUSSION THREAD

**Discussion by lilingxi01** [supabase/supabase#11065] [Questions]
I wonder if there is a documentation (site or markdown) for Supabase Swift.

Although most of APIs are the same with JS version, there are still a lot of different things that need to be noted or worth to point out.

It seems like there isn't an official one out there. So if there is no an ongoing one, I might consider to make one (probably unofficial).

**Comment by Hallidayo** [comment]
Hi @lilingxi01 - Their is a [community repo here](https://github.com/supabase-community/supabase-swift) that may help.
